### PR TITLE
zehra-dev

### PR DESCRIPTION
**Pull request by zehrakopar**
zehra-dev

### CODE REVIEW

File: js homework week2/file1.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+let turkish = 'Merhaba Dunya!';
+let english = 'Hello World!';
+let dutch = 'Hallo Werld!';
+
+console.log(turkish);
+console.log(english);
+console.log(dutch);
```
Comment (marcianoviereck) [inline js homework week2/file1.js:7]: Nicenice!

I think you forgot an 'e' though for the dutch one:

"Hallo Wereld!" :)

File: js homework week2/file10.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+console.log(typeof myArray[0]);
+console.log(typeof myArray[1]);
+console.log(typeof myArray[2]);
+
+
+
+
+let infinity1 = 6/0;
+let infinity2 = 10/0;
+if (infinity1===infinity2){console.log(true)}
```
Comment (marcianoviereck) [inline js homework week2/file10.js]: The solution is good! But your code is easier to read if you write statements like this on multiple lines (and add more spaces). For example:

```
if (infinity1 === infinity2){
  console.log(true);
} else {
  console.log(false); 
}
```

File: js homework week2/file5.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+let z = 7.25;
+console.log(z);
+let a = Math.round(z);
+console.log(a);
+let highest = Math.max(a,z);
```
Comment (marcianoviereck) [inline js homework week2/file5.js:5]: Nice! Good solution!

File: js homework week2/file6.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+let flowers = [];
+console.log('the value of my array will be: names of flowers');
```
Comment (marcianoviereck) [inline js homework week2/file6.js]: At this point in the code, the value of the array is not `names of flowers`.

File: js homework week2/file8.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+{console.log('SAME TYPE');}
+else {console.log('Don\'t have the same type');}
+
+if (typeof mySon == typeof hisSecondName) 
+{console.log('SAME TYPE');}
+else {console.log('Don\'t have the same type');}
+
+if (typeof mySon == typeof hisToys) 
+{console.log('SAME TYPE');}
+else {console.log('Don\'t have the same type');}
```
Comment (marcianoviereck) [inline js homework week2/file8.js:89]: I think you forgot to compare some variables here.

And do you think its better to use `==` or `===` ?

**Review by marcianoviereck — CHANGES_REQUESTED** [review]
The code looks good and clean! I do have some comments though

File: js homework week2/file1.js (hunk 1 above)
Comment (zehrakopar) [reply js homework week2/file1.js:7]: jaa. het is wereld! done

File: js homework week2/file6.js (hunk 4 above)
Comment (zehrakopar) [reply js homework week2/file6.js]: ok done.

File: js homework week2/file10.js (hunk 2 above)
Comment (zehrakopar) [reply js homework week2/file10.js]: ok done

File: js homework week2/file8.js (hunk 5 above)
Comment (zehrakopar) [reply js homework week2/file8.js:89]: it is corrected. I ve thought that showing a couple of example is enough.

**Review by marcianoviereck — APPROVED** [review]
Nice, the homework is approved now!:)

**Pull request closed without merging**